supertokens_python/recipe/emailverification/interfaces.py

```diff
@@ -224,6 +227,17 @@ def __init__(self):
         self.is_email_already_verified_error = True
 
 
+class TypeEmailVerificationEmailDeliveryInput:
+    def __init__(
+        self, user: User,
+        email_verify_link: str, user_context: Dict[str, Any]
```

Comment:
`user_context` should be a separate param and not as a field in this type

Reply:
todo

Reply:
Fixed.